schema/va-spec/base/pathogenicity-evidenceline-profile-source.yaml

```diff
@@ -0,0 +1,100 @@
+$schema: "https://json-schema.org/draft/2020-12/schema"
+$id: "https://w3id.org/ga4gh/schema/va-spec/1.x/base/pathogenicity-evidenceline-profile-source.yaml"
+title: Variant Pathogenicity EvidenceLine Standard Profile
+strict: true
+
+imports:
+  domain-entities: ./domain-entities-source.yaml
+  cat-vrs: ../../cat-vrs/cat-vrs-source.yaml
+  vrs: ../../vrs/vrs-source.yaml
+  gks-core: ../../gks-core/gks-core-source.yaml
+
+$defs:
+  # Pathogenicity Evidence Line
+  PathogenicityEvidenceLine:    
```

Comment:
@mbrush I'm curious if you intentionally left `Variant` out of the name of this class. We have `VariantPathogenicityStatement` and `VariantPathogenicityProposition` but this class is just `PathogenicityEvidenceLine` instead of `VariantPathogenicityEvidenceLine`? It seems inconsistent.

Reply:
Agreed.  That was just an oversight.  We should call it VariantPathogenicityEvidenceLine.